### ISSUE TITLE
Python virtual environment?

### ISSUE THREAD

**Issue by Jdruwe** [whiskerz007/proxmox_hassio_lxc#13]
This is just a simple question and not an issue as I didn't find any other way to contact you. Would an installation using https://www.home-assistant.io/docs/installation/virtualenv/ on lxc be even faster than using docker?

**Comment by whiskerz007** [comment]
TLDR; I do not believe any speed performance is perceivable by running Home Assistant in pythonenv versus Docker. You forfeit the addon system afforded to you by using Docker when you use pythonenv.

I do not have any real world data to back up my position, but I'll entertain the question using my understanding of the technology. Containers usually run in an isolated namespace, which can restrict access to resources normally accessible to applications. The applications in the container run at effectively the same speed as running them on the host since the overhead has to do with access control, not emulating hardware. Docker creates and runs containers a little differently than LXC, but the same principle applies. Overhead is mostly RAM utilization and some CPU cycles when evaluating access control permissions.

**Comment by Jdruwe** [comment]
Thx for the information, this can be closed.
